### PR TITLE
🛠️ Adding BuildGuide branch for the ease of use

### PR DESCRIPTION
**Pull request by k0gen**
These are the necessary changes for the build to produce working embassy.img

### CODE REVIEW

**Review by dr-bonez — APPROVED** [review]

File: BuildGuide.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      stack install --stack-yaml=stack-ghc-84.yaml --system-ghc
+      export PATH=~/.local/bin:$PATH
+      ```
+
+5. Clone EmbassyOS & try to *make* the `agent`:
+   1. First attempt
+      > :information_source: The first time you run **make** you'll get an error
+      
+      ```
+      sudo apt install llvm-9 libgmp-dev
```
Comment (elvece) [inline BuildGuide.md:76]: fwiw I needed to run export PATH=/usr/lib/llvm-9/bin:$PATH to resolve errors here.

Comment (k0gen) [reply BuildGuide.md]: Although I didn't have this kind of issue it won't hurt adding it to the guide.
Done!

Comment (k0gen) [reply BuildGuide.md]: Although I didn't have this kind of issue it won't hurt adding it to the guide.
Done!

**Review by k0gen — COMMENTED** [review]
I have just followed the guide again on a fresh system all went well 👌

**Review by dr-bonez — APPROVED** [review]

**Review by elvece — APPROVED** [review]

**Pull request merged**